src/FrankaLightWeightInterface.cpp

```diff
@@ -171,6 +227,46 @@ void FrankaLightWeightInterface::run_joint_torques_controller() {
   }
 }
 
+void FrankaLightWeightInterface::run_cartesian_velocities_controller() {
+  // Set additional parameters always before the control loop, NEVER in the control loop!
+
+  this->franka_robot_->setCartesianImpedance({{100, 100, 100, 10, 10, 10}});
+
+  // Set collision behavior.
+  this->franka_robot_->setCollisionBehavior(
```

Comment:
I think at some point we should find a way to parameterize more those hardcoded gains. Probably a config file would suffice if that is possible.